packages/mobile/src/components/status-message/StatusMessage.tsx

```diff
@@ -10,6 +10,7 @@ type MessageStatus = 'default' | 'valid' | 'error'
 type StatusMessageProps = {
   label: string
   status: MessageStatus
+  style?: StyleProp<ViewStyle>
 }
```

Comment:
For consistency let's do:

```suggestion
type StatusMessageProps = {
  label: string
  status: MessageStatus
} & StylesProps<{
  root?: ViewStyle
}>
```

And then make sure to apply both the `stylesProp` and `style` props to the root:

```ts
export const StatusMessage = ({ label, status, style, styles: stylesProp }: StatusMessageProps) => {
```

```ts
<View style={[styles.root, style, stylesProp.root]}>
```